release-notes/v1_107.md

```diff
@@ -14,26 +14,18 @@ _Release date: December 10, 2025_
 <!-- DOWNLOAD_LINKS_PLACEHOLDER -->
 
 ---
-
-Welcome to the November 2025 release of Visual Studio Code.
+VS Code now includes Agent HQ, background agents and custom agents for your org.
+- **Agent HQ**: You can use GitHub Copilot alongside your other coding agents to complete multiple tasks. Leverage background, cloud, or local agents and their relative strengths for the right job, all at the same time. 
+- **Background agents** work in their own isolated workspaces, managed for you by VS Code.
+- **Custom agents for organizations:** Your team can now share custom agents at the org level.
 
 ![VS Code November 2025 release image.](images/1_107/release-highlights.webp)
 
-This release brings significant updates across these key areas:
-
-* **Agent HQ** removes the boundaries between local, background, and cloud agents, seamlessly delegating tasks to the best-suited agent.
-* **Security and Governance** let you keep agents in check and stay in control with enterprise policies
-* **Stay productive** by giving you the information and tools you need, with the least friction
-
 Happy Coding!
 
 <br>
 
->If you'd like to read these release notes online, go to [Updates](https://code.visualstudio.com/updates) on [code.visualstudio.com](https://code.visualstudio.com).<br>
```

Comment:
We use these release notes in-product, so we want to keep the option for people to view them on the web.